content/en/continuous_integration/pipelines/buildkite.md

```diff
@@ -83,6 +83,17 @@ The [Pipelines][3] and [Pipeline Executions][4] pages populate with data after t
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+### Partial and downstream pipelines
+In the `Pipeline Executions` page, you can now use the filters below in the search bar:
+* `Downstream Pipeline` - possible values: `true`, `false` 
+* `Manually Triggered` - possible values: `true`, `false` 
+* `Partial Pipeline` - possible values: `retry`, `paused`, `resumed`
+
+{{< img src="ci/partial_retries_search_tags.png" alt="GitLab Partial Retry Filters" style="width:100%;">}}
```

Comment:
```suggestion
{{< img src="ci/partial_retries_search_tags.png" alt="The Pipeline executions page with Partial Pipeline:retry entered in the search query" style="width:100%;">}}
```

Reply:
Just following up on this image:
- Is the Pipeline Executions page the correct place to enter this search? The UI of that page doesn't seem to match your screenshot. Perhaps this is a limitation of the demo account?
- I updated the alt text with our internal guidelines in mind

Reply:
Yes, the Pipeline Executions page is the correct place for the search, but the demo account does not contain a lot of our newest features. Updating our demo environment is something we are actively working on :)